Resources/doc/index.rst

```diff
@@ -348,5 +348,10 @@ class (as shown above), you can access it in the ``load()`` method.
     If you are too lazy to implement the needed method :method:`Symfony\\Component\\DependencyInjection\\ContainerInterface::setContainer`,
     you can then extend your class with :class:`Symfony\\Component\\DependencyInjection\\ContainerAware`.
 
+Hautelook/AliceBundle
+---------------------
+
+You can also use the popular [hautelook/AliceBundle](https://github.com/hautelook/AliceBundle) bundle to manage your fixtures with [nelmio/alice](https://github.com/nelmio/alice) and [fzaninotto/Faker](https://github.com/fzaninotto/Faker).
```

Comment:
to be consistent, you should make these links with footnotes as well (see DoctrineFixturesBundle in this file for an example).